pkg/render/node.go

```diff
@@ -1055,15 +1057,12 @@ func (c *nodeComponent) nodeVolumes() []corev1.Volume {
 		volumes = append(volumes, corev1.Volume{Name: "cni-log-dir", VolumeSource: corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{Path: cniLogDir}}})
 	}
 
-	// Override with Tigera-specific config.
-	if c.cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise {
```

Comment:
Not required but I wonder if we should replace this if check to see if LogCollector exists? Because we don't need this unless the LogCollector is enabled.